lib/facility_factory.rb

```diff
@@ -0,0 +1,89 @@
+class FacilityInfo
+    attr_reader :geom, :id, :office, :address_1, :address_2,
```

Comment:
So in coding, we have this idea of only coding what we need at that moment. Which means not including all these attributes. We only want to bring in the attributes that we need to create a Facility, so looking back at the Facility class, there are only a handful that we need. Most of these can be left out.